apps/console/src/features/applications/components/application-list.tsx

```diff
@@ -378,28 +377,34 @@ export const ApplicationList: FunctionComponent<ApplicationListPropsInterface> =
                 key: "inboundKey",
                 render: (app: ApplicationListItemInterface): ReactNode => {
                     let inboundAuthKey: string = "";
-                    let inboundAuthType: ApplicationInboundTypes;
+                    let inboundAuthType: string = "";
+                    let inboundAuthTypeLabelClass: string = "";
 
                     if (app.clientId) {
                         inboundAuthKey = app.clientId;
-                        inboundAuthType = ApplicationInboundTypes.CLIENTID;
+                        inboundAuthType = "Client ID";
```

Comment:
Any reason for removing the `Enum` here?

Reply:
It's a mistake. I put enums back.